src/ClassicUO.Client/Game/Managers/MacroManager.cs

```diff
@@ -2016,6 +2050,24 @@ public void Load(XmlElement xml)
                     PushToBack(m);
                 }
             }
+
+            XmlElement buttons = xml["controllerbuttons"];
+
+            if (buttons != null)
+            {
+                List<SDL.SDL_GameControllerButton> savedButtons = new List<SDL_GameControllerButton>();
+                foreach (XmlElement buttonNum in buttons.GetElementsByTagName("button"))
+                {
+                    if (int.TryParse(buttonNum.InnerText, out int b))
+                    {
+                        if (Enum.IsDefined(typeof(SDL_GameControllerButton), b))
```

Comment:
Use this https://learn.microsoft.com/en-us/dotnet/api/system.enum.isdefined?view=net-9.0#system-enum-isdefined-1(-0) to avoid reflection